### ISSUE TITLE
Why it was 2 years we have no update? is it obsolete?

### ISSUE THREAD

**Issue by trumhemcut** [vergecurrency/nodejs-verge#7]

**Comment by juanca** [comment]
Does it work with Stealth addresses?

**Comment by justinvforvendetta** [comment]
@trumhemcut no, it just didnt need anything else because it worked perfectly.

**Comment by justinvforvendetta** [comment]
@juanca it will soon, its very easy to add the rpc codes for it. should only take a few minutes.

**Comment by benammann** [comment]
@juanca as you can see in my pull request https://github.com/vergecurrency/nodejs-verge/pull/6 i've added the missing commands

waiting to merge :)
